products/dotnet.md

```diff
@@ -25,15 +26,15 @@ releases:
     lts: false
     releaseDate: 2022-11-08
     eol: 2024-05-14
-    latest: "7.0.5"
-    latestReleaseDate: 2023-04-11
+    latest: "7.0.7"
```

Comment:
LGTM, 7.0.6 not listed on https://github.com/dotnet/core/tree/main/release-notes/7.0.